extension/js/common/platform/util.ts

```diff
@@ -32,3 +32,50 @@ export const moveElementInArray = <T>(arr: Array<T>, oldIndex: number, newIndex:
   arr.splice(newIndex, 0, arr.splice(oldIndex, 1)[0]);
   return arr;
 };
+// tslint:disable:no-unsafe-any
+export const compareObjects = (object1: { [key: string]: any }, object2: { [key: string]: any }) => {
```

Comment:
Related to another comment, this whole section can be removed.

Also, this file should only contain platform-specific code. The kind of code that would be written differently for each platform (browser vs node).

It says it right above:
```js
/**
 * Functions which must be written differently to run in NodeJS versus in web browsers.
 *
 * If the code would be the same on both platforms, it does not belong here (or anywhere in platform/ directory)
 */
```

Reply:
Please take a look at my comment above and if I can leave this method in Flowcrypt I can move it to another file.